### PR TITLE
Add LEGO

### PR DESCRIPTION
**Pull request by johntheholman**
### Description:
- Added JSON file for website and how to connect 2FA
- Added SVG file for LEGO

### SimilarWeb:
- Global Rank: 769
- Category: Games

### Screenshots (because there was no Documentation):
Screen Shot 2023-01-29 at 11 23 02 PM

### CODE REVIEW

File: entries/l/lego.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "LEGO": {
+    "domain": "lego.com",
+    "tfa": [
+      "email"
+    ],
+    "notes": "To turn on, Go to Account Overview > Personal & Address Details > LEGO Account Details > Security > Use two-factor authentication",
```
Comment (hkamran80) [inline entries/l/lego.com.json]:
```suggestion
```

None of our other sites have the instructions in the notes

File: entries/l/lego.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "categories": [
+      "creativity",
```
Comment (hkamran80) [inline entries/l/lego.com.json]:
```suggestion
```

This doesn't fit with the other `creativity` sites

File: img/l/lego.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...4.077-6.04.347-15.339.679-20.902.285 0 0-4.707 7.171-9.71 19.952 26.197-3.688 37.306 2.258 32.884 17.412-5.986 20.472-23.723 21.82-45.445 17.932zm128.85-63.695c-7.084 0-11.718 4.487-15.165 9.331-7.593 10.664-24.045 53.396-26.606 70.95-1.756 12.022 3.436 14.47 9.11 14.47 9.104 0 19.31-9.63 22.792-25.518 0 0-17.329-.422-12.553-15.79 4.642-14.912 13.449-18.142 28.047-18.741 28.76-1.174 25.92 20.06 23.619 31.385-7.475 36.775-33.647 66.307-70.668 66.307-25.345 0-41.078-14.02-41.078-39.864 0-18.416 9.136-47.327 16.38-64.244 15.414-36.01 31.58-61.21 72.766-61.21 24.71 0 44.199 8.88 41.341 32.028-2.095 16.99-10.599 26.967-26.046 28.174-4.32.336-21.996-.112-16.133-16.834 2.04-5.849 2.886-10.444-5.806-10.444zm155.544 30.392c-4.353 21.682-14.147 47.327-24.778 66.44-17.344 31.187-38.412 35.555-56.8 35.334-18.398-.22-39.123-7.012-39.282-35.384-.112-20.37 8.674-49.122 16.134-67.969 13.001-34.167 26.288-62.026 68.29-61.524 48.982.582 40.247 44.141 36.436 63.103zm-41.551-29.68c-4.205-.058-7.875.766-11.173 5.852-7.41 9.598-29.07 68.789-28.687 80.594.137 4.241 2.481 7.951 7.705 7.951 5.993.008 9.324-4.136 12.268-9.135 6.817-11.535 25.985-63.855 26.512-76.935.156-3.812-.48-8.251-6.625-8.327z"/></svg>
```
Comment (hkamran80) [inline img/l/lego.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="#fff" d="M3.5 3.4v505.2h505.1V3.4H3.5z"/><path fill="#f6ec36" d="M3.5 3.4v505.2h505.1V3.4H3.5zm471.5 210c-.8 15.7-5.8 35.4-10.2 48.5-17.8 52.7-38.4 85.5-86 85.5-14.1 0-39-3.8-48.2-28.8l-2.2-5.9-3.9 5a77.7 77.7 0 0 1-60.8 29.9c-18.2 0-32.8-6-42.3-17.7l-2.6-3.1-2.7 3c-10.1 11.2-27.7 17.6-48.3 17.6a52.4 52.4 0 0 1-39.3-15.5l-2.5-2.7-2.6 2.6c-10.2 10.3-26.3 15.5-46.5 15-24.5-.6-39.5-14.2-40-36.6-.7-34.5 32.7-107 46.5-129a41.5 41.5 0 0 1 37.7-22c11.2 0 18.8 2.4 23.3 7.2 4.2 4.4 5 8 5.4 17l.4 11.4 6-9.7c14.7-23.6 39-26.9 63-26.9 16.8 0 31.4 6.2 37.2 15.7l2.2 3.5 3.1-2.7a75.7 75.7 0 0 1 49.2-16.3c21.2 0 36.5 5.2 45.7 15.6 2 2.3 3.4 4 5.4 8l2.3 5 3.5-4.2a67.9 67.9 0 0 1 54.5-24c18 0 31.6 4.9 40.5 14.3 11.3 11.8 12.9 28.4 12.2 40.3z"/><path fill="#d01012" d="M3.5 3.4v505.2h505.1V3.4H3.5zm485.8 229.4a289 289 0 0 1-21.4 66c-17.7 36.5-41.3 64.6-87 64.6-24.3 0-44.3-7.8-56-22.7a91.7 91.7 0 0 1-61.3 22.9c-17.6 0-33.2-5.1-45-14.5a87.6 87.6 0 0 1-50.4 14.4c-16.4 0-31-4.5-42.6-12.8a84.2 84.2 0 0 1-49 12.2c-32.7-.8-55.2-21.5-55.9-51.5-.8-40 32.8-113 48.6-138.3 11.9-19.5 30-30 52.2-30 24 0 32.7 6.9 37 15.6 18.6-15.7 43.2-16.6 60.8-16.6 19.3 0 31.5 5 41.8 13.3a94.8 94.8 0 0 1 50-13c25 0 43.5 5.8 56.4 18.6 15-12.6 34-18.3 55-18.3 28 0 46.3 10 57 25 15.3 21.3 12.6 42.6 9.8 65z"/><path d="M0 0v512h512V0H0zm505 7v498H7V7h498z"/><path d="M486.8 153.6v-3.4h2.4c1.7 0 2.6.6 2.6 1.7 0 .9-.6 1.7-2.3 1.7h-2.7zm8.5 7.3-1.5-2.6c-1.3-2.2-1.6-2.6-2.7-3 2.1-.3 3.4-1.7 3.4-3.6 0-2-1.3-3.5-4-3.5h-6.3v12.7h2.6v-5.2h.5c1.1 0 1.6.1 2 .6.6.5 1.3 1.6 1.8 2.5l1 2h3.2zm-6.3-16.3a9.9 9.9 0 1 1 0 19.8 9.9 9.9 0 0 1 0-19.8zm0-2.4a12.2 12.2 0 1 0 0 24.5 12.2 12.2 0 0 0 12.3-12.2 12.2 12.2 0 0 0-12.3-12.3zm-66.7 13.1a71.2 71.2 0 0 0-57.2 25.2c-2-4.4-3.6-6.3-5.9-8.8-10.1-11.5-26.4-16.8-48.3-16.8a78.8 78.8 0 0 0-51.5 17.1c-6.4-10.4-21.7-17.3-40.2-17.3-24 0-50.3 3.4-66 28.5-.3-9-1.1-13.8-6.2-19.2-6.2-6.5-16.2-8.2-26-8.2-17.7 0-31 7.7-40.6 23.5-14 22.3-47.7 95.5-47 131 .5 22.9 15.9 39.4 43.4 40 21.5.6 38.4-5.2 49.1-16a55.7 55.7 0 0 0 41.9 16.6c19.6 0 39.3-5.9 50.9-18.7 9.8 12 25.2 19 45 19 26-.2 48.7-12.5 63.6-31.3 8.1 22.2 28.8 31 51.4 31 49.6 0 71.3-34 89.4-87.9 4.1-12.1 9.6-32.4 10.5-49.4 1.5-28.3-10.5-58.3-56.3-58.3zM92.7 290.7c25.7-4.4 32.1 4.8 31.3 13.4-2.4 25.7-26 31.4-46.4 31-14.9-.4-28.2-7.2-28.6-24.4-.6-30.2 30.1-99.9 44.6-123 6.7-11 14.8-16.4 27.8-16.4 12.3 0 15.3 6.3 15.1 14-.4 20.1-33 81.5-43.8 105.4zm83.5-23.6c-1.7 4.5-5 14.1-7.5 24.3a83.5 83.5 0 0 1 25.5-3.3c12.3.3 20.2 5.4 20.2 15.6 0 24.7-27.3 32-46.2 32-20.8 0-39-11.9-39-34.6 0-26.6 14.4-67 28-94 16.6-33 33.6-37.4 62.9-37.4 12.8 0 27.7 5.5 27.7 17.7 0 16.8-14.3 23.3-28.4 24-6 .4-15.4.7-21 .4 0 0-4.7 7.1-9.7 20 26.2-3.8 37.3 2.2 33 17.3-6 20.5-23.8 21.8-45.5 18zM305 203.4c-7 0-11.7 4.4-15.1 9.3-7.6 10.7-24 53.4-26.6 71-1.8 12 3.4 14.4 9 14.4 9.2 0 19.4-9.6 22.9-25.5 0 0-17.4-.4-12.6-15.8 4.7-15 13.5-18.1 28-18.7 28.8-1.2 26 20 23.7 31.3-7.5 36.8-33.7 66.4-70.7 66.4-25.3 0-41-14-41-40 0-18.3 9-47.2 16.3-64.2 15.4-36 31.6-61.2 72.8-61.2 24.7 0 44.2 9 41.3 32-2 17-10.6 27-26 28.2-4.3.4-22 0-16.2-16.8 2-5.8 3-10.4-5.8-10.4zm155.6 30.3a248.5 248.5 0 0 1-24.8 66.5c-17.3 31.2-38.4 35.5-56.8 35.3-18.4-.2-39.1-7-39.3-35.4-.1-20.3 8.7-49 16.2-68 13-34.1 26.2-62 68.2-61.5 49 .6 40.3 44.2 36.5 63.1zM419 204.1c-4.2 0-7.8.7-11.1 5.8-7.5 9.6-29.1 68.8-28.7 80.6.1 4.3 2.4 8 7.7 8 6 0 9.3-4.2 12.2-9.2 6.9-11.5 26-63.8 26.6-76.9.1-3.8-.5-8.3-6.7-8.3z"/></svg>
```

Here's a minified version

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/l/lego.com.json (hunk 1 above)
Comment (johntheholman) [reply entries/l/lego.com.json]: Good to know. I understand the notes section now and where it appears on the website. Will update any other PRs in the future accordingly.

File: entries/l/lego.com.json (hunk 2 above)
Comment (johntheholman) [reply entries/l/lego.com.json]: I am still trying to figure out the categories so thank you for the feedback here.

File: img/l/lego.com.svg (hunk 3 above)
Comment (johntheholman) [reply img/l/lego.com.svg]: Thanks!

File: entries/l/lego.com.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "LEGO": {
+    "domain": "lego.com",
+    "url": "https://www.lego.com/en-us",
```
Comment (hkamran80) [inline entries/l/lego.com.json]:
```suggestion
```

You only need the `url` key if going to the base domain, that is `https://[domain]`, fails

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/l/lego.com.json (hunk 4 above)
Comment (johntheholman) [reply entries/l/lego.com.json]: Thank you for the explanation.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**